typescript/infra/config/environments/testnet4/agent.ts

```diff
@@ -225,6 +228,20 @@ const scraperResources = {
   },
 };
 
+const defaultIsmCacheConfig: IsmCacheConfig = {
+  // Default ISM Routing ISMs change configs based off message content,
+  // so they are not specified here.
+  moduleTypes: [
+    ModuleType.AGGREGATION,
+    ModuleType.MERKLE_ROOT_MULTISIG,
+    ModuleType.MESSAGE_ID_MULTISIG,
```

Comment:
do we care about the weighted versions of these for now or nah?

Reply:
Good q, atm we don't have the weighted functionality in the relayer yet so we don't care